netctl/netctl.go

```diff
@@ -252,6 +252,71 @@ func listNetworks(ctx *cli.Context) {
 	}
 }
 
+func createTenant(ctx *cli.Context) {
+	argCheck(1, ctx)
+
+	tenant := ctx.Args()[0]
+	subnetPool := ctx.String("subnet-pool")
+	subnetLen := ctx.Int("subnet-len")
+	vlans := ctx.String("vlans")
+	vxlans := ctx.String("vxlans")
+
+	if subnetPool == "" || vlans == "" || vxlans == "" {
```

Comment:
do we need a check for `subnet-len`?

On a related note, may be these validations should be be left to objmodel and it's client interface? May be it is already planned as we are moving towards a shared objmodel with code generation. Just want to confirm my understanding.